streams/src/test/java/org/apache/kafka/streams/state/internals/ChangeLoggingKeyValueBytesStoreTest.java

```diff
@@ -196,6 +201,26 @@ public void shouldReturnValueOnGetWhenExists() {
         assertThat(store.get(hello), equalTo(world));
     }
 
+    @Test
+    public void shouldGetRecordsWithPrefixKey() {
+        store.put(hi, there);
+        store.put(Bytes.increment(hi), world);
+        final KeyValueIterator<Bytes, byte[]> keysWithPrefix = store.prefixScan("hi", new StringSerializer());
```

Comment:
```suggestion
        final KeyValueIterator<Bytes, byte[]> keysWithPrefix = store.prefixScan(hi.toString(), new StringSerializer());
```
In such a way, you can reuse variable `hi` and `there`. Similar is true for my suggestions below.